src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Flex } from "@chakra-ui/react"
+
+export interface BannerProps {
+  children: string
```

Comment:
i think this signature should be amended to `ReactNode` as we are using it below with a component (`ChakraLink`).

design system types it as string as the they want it to go through the `useMarkdown` hook to generate the appropriate html, which we are not providing here.

Reply:
This will be addressed when we entirely replace the custom Banner component with the design system one, so I'm ok overlooking it for now